app.py

```diff
@@ -1,30 +1,37 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from api_items import getItemData, createItem, updateItem, removeItem, getStatusIdForTitle
+import sys
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
```

Comment:
Well done removing SECRET_KEY from .env.template, but you also need to delete its usage or the app crashes - you can delete this line and the flask_config.py file.

Reply:
Line and file removed in [7e28c8a](https://github.com/rysharprules/DevOps-Course-Starter/pull/2/commits/7e28c8a1767f76bb8d9b6ca560c7d6ce52e6b78e)